src/js/components/Person.js

```diff
@@ -65,6 +77,26 @@ export default class Person extends Component {
     Rest.get('/ldap/', params).end(this._onPersonResponse);
   }
 
+  _onLocationResponse (err, res) {
+    if (err) {
+      this.setState({location: {}, error: err});
+    } else if (res.ok) {
+      const result = res.body;
+      this.setState({location: result[0], error: null}, () => {
+        this._renderTimezoneOffset();
```

Comment:
Why are we calling a _render method as part of a rest response? I would think the setState would cause the component to re-render, which would include the timezone offset.